packages/block-library/src/file/edit.js

```diff
@@ -297,7 +308,10 @@ function FileEdit( { attributes, isSelected, setAttributes, clientId } ) {
 										'button'
 									)
 								) }
-								value={ downloadButtonText }
+								value={
+									downloadButtonText ??
+									_x( 'Download', 'button label' )
```

Comment:
Noting that this means that the translated `Download` text will no longer get serialized in the saved content.